nouvelles-perspectives-en-sciences-sociales.csl

```diff
@@ -524,7 +526,7 @@
                 <text value="loc. cit." font-style="italic"/>
               </if>
               <else-if type="book report thesis chapter entry-encyclopedia entry-dictionary" match="any">
-                <text value="op. cit." font-style="italic"/>
+                <text value="op.&#160;cit." font-style="italic"/>
```

Comment:
approved

Reply:
This modification will not affect our proposition ? We need a bibliography for our journal as the sample citations that csl-bot sent us. The citations generated and based on our proposed changes.